stdlib/public/stubs/GlobalObjects.cpp

```diff
@@ -106,12 +106,37 @@ swift::_SwiftEmptySetStorage swift::_swiftEmptySetStorage = {
   0 // int entries; (zero'd bits)
 };
 
+#if defined(__Fuchsia__)
+#include <zircon/syscalls.h>
+static __swift_uint64_t randomUInt64() {
+  __swift_uint64_t returnval;
+  size_t remaining = sizeof(__swift_uint64_t);
+  unsigned char* offset = reinterpret_cast<unsigned char*>(&returnval);
+  size_t actual;
+  size_t read_len;
+  do {
+    // We can only read a limited number of bytes via the syscall at a time.
+    read_len =
+        remaining > ZX_CPRNG_DRAW_MAX_LEN ? ZX_CPRNG_DRAW_MAX_LEN : remaining;
+
+    zx_status_t status = zx_cprng_draw(offset, read_len, &actual);
+
+    // decrement the remainder and update the pointer offset in the buffer
```

Comment:
Please start comments with a capital letter and end with a period (per LLVM style guide) (here and everywhere else in the PR).